backend/src/news/router.py

```diff
@@ -4,10 +4,10 @@
 from ..auth.dependencies import session_opener, authenticate_user_token
 from ..utils import _id_counter
 
-from .utils import toggle_upvote, fetch_news_upvote_details, fetch_latest_news_info, udn_crawler
+from .utils import toggle_upvote, fetch_news_upvote_details, fetch_latest_news_info, udn_crawler, anthropic_client
 from ..models import NewsArticle
-from .schema import PromptRequest, NewsSummaryRequestSchema
-from .utils import llm_client
+from .schema import PromptRequest, NewsSummaryRequestSchema, NewsSummaryCustomModelRequestSchema
+from .utils import openai_client
```

Comment:
Nitpicking here, I just think it makes more sense to put anthropic_client and openai_client on the same line
from . utils import anthropic_client, openai_client